tests/hypertext/HTMLLintBearTest.py

```diff
@@ -27,3 +27,10 @@
     invalid_files=(test_file,),
     settings={'htmllint_ignore': 'quotation'},
     tempfile_kwargs={'suffix': '.html'})
+
+HTMLLintBearDisableCheckerTest = verify_local_bear(
+    HTMLLintBear,
+    valid_files=(test_file,),
+    invalid_files=(),
+    settings={'optional_tag': 'off'},
```

Comment:
I think it does see https://github.com/sk-/html-linter/blob/master/html_linter.py#L1083 ..